compliance/collection/nodeinventorizer/caching_inventorizer.go

```diff
@@ -35,6 +35,11 @@ type inventoryWrap struct {
 	CachedInventory      string    // Serialized form of the cached inventory
 }
 
+type cacheState struct {
+	CachedInventory *storage.NodeInventory
+	BackoffDuration time.Duration
+}
```

Comment:
[nit] No need to export these fields since they are internal and used only in this package.
```suggestion
type cacheState struct {
	cachedInventory *storage.NodeInventory
	backoffDuration time.Duration
}
```
Also, since this is internal, you may consider shorten their names to be able to create a struct in one line. E.g. instead of
```go
		return &cacheState{
			CachedInventory: nil,
			BackoffDuration: c.maxBackoff,
		}
```
do
```go
		return &cacheState { cache: nil, backoff: c.maxBackoff }
```
Up to you.